plugins/inputs/kafka_consumer/kafka_consumer_test.go

```diff
@@ -234,7 +234,7 @@ func (c *FakeConsumerGroupClaim) InitialOffset() int64 {
 	panic("not implemented")
 }
 
-func (c *FakeConsumerGroupClaim) HighWaterMarkOffset() int64 {
+func (c *FakeConsumerGroupClaim) HiWaterMarkOffset() int64 {
```

Comment:
I'm dropping this (as `cg.ConsumeClaim` uses `HighWaterMarkOffset`).

The reason for the change is that this repository mostly uses `hiwater`, e.g. `hiwater_bytes` and `lowater_bytes`, but as this is an API thing from an upstream, it's just one of those inconsistencies that a project will have.

(The other fun ones are `marshal`/`marshall` -- this project is almost entirely in the former, and I'm trying to make that consistent and `falsy`/`falsey` -- neither of which are used by this project.)